client/src/ui/css/App.css

```diff
@@ -81,14 +81,8 @@ nav {
   background-position: bottom; 
 }
 
-.full-height{
-  min-height: 100.1%; /* Needed so that background is flush against bottom */
-  height: 100.09%;
-}
-
-
 .background-image{
-  background: url('backgroundNew-edited.jpg') fixed no-repeat;
+  background: url('/backgroundNew-edited.jpg') fixed no-repeat;
   background-size: cover;
   object-fit: contain;
 }
```

Comment:
Let's avoid doing css change here since it might mess up with the redesign branch.
https://github.com/cornell-dti/course-reviews-react-2.0/tree/blue-redesign
@julian-londono the redesign branch should probably be merged into master ASAP. All future pull requests should target `master` instead of a random branch.

Reply:
Ah, so Bootstrap v3 was an absolute pain to get to build correctly on the client so I upped to v4 as a part of this.

The URL change is necessary due to create-react-app changes.